src/lib.rs

```diff
@@ -40,7 +40,14 @@ extern crate webpki_roots;
 
 mod api;
 mod batch;
+
+#[cfg(any(
+    feature = "default",
```

Comment:
Not 100% sure about this, but I think this line can be removed, because `default` implies `proxy` and `use-rustls`, which should be covered by the lines below

Reply:
it looks it works without `default`, done